metadata.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: irc-bridge
+display-name: IRC Bridge
+summary: IRC Bridge charm for Matrix.
+docs: https://discourse.charmhub.io/
+issues: https://github.com/canonical/irc-bridge-operator/issues
+maintainers:
+  - https://launchpad.net/~canonical-is-devops
+source: https://github.com/canonical/irc-bridge-operator
+
+description: |
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
```

Comment:
DNS?

Reply:
^fixed, good catch! Seems the issue is always DNS..